tests/test_generate_success.py

```diff
@@ -0,0 +1,54 @@
+from gatorgrade.generate.generate import generate_config
+import pytest
```

Comment:
These test cases all look good! However, I just want to point out that there's a Pylint error saying that you should import pytest at the top before the importing of generate_config.

Reply:
Thank you for the review!